plugins/processors/multiplier/README.md

```diff
@@ -0,0 +1,30 @@
+# Multiplier Processor
+
+The multiplier processor is used to change the values of metrics with some multiplication factor.
+
+### Examples:
+
+* [input.system](./plugins/inputs/system) plugin produces metric [uptime](./plugins/inputs/system/SYSTEM_README.md#metrics) with amount of seconds after computer start. If it is needed to get value in hours multiplier configuration should contain
+"system uptime=0.00027777777"
+
+* Some **metric** contains **value** in range (0;&nbsp;1). If it is needed to get values in more human readable percentage format multiplier configuration should contain
+"metric value=100"
+
+**Note:** Types of changed values stay original.
+
```

Comment:
It's a matter of taste for the most part, but I would've phrased these two notes this way:
`Note: Multiplied values keep thier original types.`
`Note: Values that cannot be multiplied remain unchanged.`

Reply:
Thanks. Fixed.